cmd/node/main.go

```diff
@@ -542,6 +552,70 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 		log.Debug("no epoch db found in storage", "error", errNotCritical.Error())
 	}
 
+	epochFoundInStorage := errNotCritical == nil
+
+	shouldCallEpochStartDataProvider := bootstrap.ShouldSyncWithTheNetwork(
+		startTime,
+		epochFoundInStorage,
```

Comment:
epochFoundInStorage is 0 when storage prunning is off.

Reply:
you're right. what do you suggest in order to get the current stored epoch if storage pruning is disabled ?